stores/mongodb/src/storage/domains/agents/index.ts

```
@@ -369,6 +413,151 @@ export class MongoDBAgentsStorage extends AgentsStorage {
+          ],
+          as: 'latestVersion',
+        },
+      });
+
+      // Add computed field for the version to use
+      pipeline.push({
+        $addFields: {
+          version: {
+            $cond: {
+              if: { $gt: [{ $size: '$activeVersion' }, 0] },
+              then: { $arrayElemAt: ['$activeVersion', 0] },
```

Biome (2.3.13)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)